src/lib/formats/tzx_cas.cpp

```diff
@@ -923,6 +1065,13 @@ static const cassette_image::Format cdt_cassette_format =
 	cdt_cassette_load,
 	nullptr
 };
+static const cassette_image::Format tsx_cassette_format =
```

Comment:
Please use consistent spacing with the rest of the code.

Reply:
I have erased the blank lines, I don't know if you mean that.